app/themes/base.js

```diff
@@ -46,25 +45,42 @@ export default {
     '16rem', // [7]
     '32rem' // [8]
   ],
-  fontSizes: [
+  fontSizes: {
     // typographic scale because
     // http://spencermortensen.com/articles/typographic-scale/
     // and
     // https://blog.madewithenvy.com/responsive-typographic-scales-in-css-b9f60431d1c4
-    '0.8rem', // [0] 10pt
-    '0.95rem', // [1] 11pt
-    '1rem', // [2] 12pt
-    '1.2rem', // [3] 14pt
-    '1.4rem', // [4] 16pt
-    '1.5rem', // [5] 18pt
-    '1.7rem', // [6] 21pt
-    '2rem', // [7] 24pt
-    '3rem', // [8] 36pt
-    '4rem', // [9] 48pt
-    '5rem', // [10] 60pt
-    '6rem', // [11] 72pt
-    '8rem' // [12] 96pt
-  ],
+    '-5': '0.50000em',
+    '-4': '0.57435em', 
+    '-3': '0.65975em',
+    '-2': '0.75785em',
+    '-1': '0.87055em',
+    0: '1.00000em',
```

Comment:
@ahdinosaur i'd find it really helpful to retain some of the info that was there previously showing how rem/em font sizes correspond to regulat pt font sizes, can you add that in please?

Reply:
okay, i'll try to add direct conversion values from http://pxtoem.com/. i'm worried they will be misleading though, since an em value is multiplied by the parent em values, so if we do this properly (as described in the article which describes the pentatonic scale we are using) then the em values will almost never match the direct conversion values.